cypress/e2e/functional/tile_tests/geometry_tool_spec.js

```diff
@@ -246,6 +254,77 @@ context('Geometry Tool', function () {
     geometryToolTile.chooseLabelOption('none');
     geometryToolTile.getGraphPointLabel().contains('AB').should('not.exist');
     geometryToolTile.getGraphPointLabel().contains('5').should('not.exist');
+    geometryToolTile.clickGraphPosition(0, 0); // deselect the segment
+
+    // Test keyboard functions to move the selected point(s)
+    cy.log('Test keyboard functions to move polygon points');
+
+    // turn on 90 degree angle for check
+    clueCanvas.clickToolbarButton('geometry', 'select');
+    geometryToolTile.selectGraphPoint(10, 5); // this point is a 90 degree angle
+    clueCanvas.clickToolbarButton('geometry', 'label');
+    geometryToolTile.toggleAngleCheckbox();
+    geometryToolTile.getGraphPointLabel().contains('90°').should('exist');
+
+    // Select the graph point at (5, 5)
+    clueCanvas.clickToolbarButton('geometry', 'select');
+    geometryToolTile.clickGraphPosition(5, 5);
+
+    // Verify that the point has been selected
+    geometryToolTile.getSelectedGraphPoint().should('have.length', 1);
+
+    // Store the original coordinates for comparison
+    let originalCcx, originalCcy;
+    geometryToolTile.getSelectedGraphPoint().then(($point) => {
+        originalCcx = parseFloat($point.attr('cx'));
+        originalCcy = parseFloat($point.attr('cy'));
+    });
+
+    // Move the selected point up using the arrow key
+    clueCanvas.clickToolbarButton('geometry', 'select');
+    geometryToolTile.clickGraphPosition(5, 5); // shared point
+    geometryToolTile.getSelectedGraphPoint().trigger('keydown', { keyCode: 38 }); // simulate up arrow key press
+
+    // Move the selected point right using the arrow key
+    geometryToolTile.getSelectedGraphPoint().trigger('keydown', { keyCode: 39 }); // simulate right arrow key press
+
+    // Verify that the point has moved: cx should be greater and cy should be less than the original values
+    geometryToolTile.getSelectedGraphPoint().then(($point) => {
+        const newCx = parseFloat($point.attr('cx'));
+        const newCy = parseFloat($point.attr('cy'));
+
+        expect(newCx).to.be.greaterThan(originalCcx);
+        expect(newCy).to.be.lessThan(originalCcy);
+    });
+
+    // Also check that the angle label has changed from its original value
+    geometryToolTile.getAngleAdornment().should(($label) => {
+        const angleText = $label.text();
+        expect(angleText).not.to.equal('90'); // 90° was the original value
```

Comment:
Shouldn't this be:
expect(angleText).not.to.equal('90°');
including the degree sign (°), or am I missing something? Alternatively, you could use ".not.contain('90')".

Reply:
Does not appear that this was resolved.
The matching test before the point was moved (line 268) is 
   `geometryToolTile.getGraphPointLabel().contains('90°').should('exist');`
So this should be the opposite of that, eg
   `expect(angleText).not.contains('90°');`

Reply:
I'll go ahead and check in this fix since it was tiny & I wanted to test it anyway.